### PR TITLE
feat: adds support for providing a proofDocument with multiple proofs

### PR DESCRIPTION
**Pull request by kdenhartog**
see issue #79 for additional details

## Description

- [X] Tests for the changes have been added (for bug fixes / features)
- [X] The commit message(s) follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [X] Changes follow the **[contributing](../docs/CONTRIBUTING.md)** document.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Which merge strategy will you use?

- [X] Squash
- [ ] Rebase (REVIEW COMMITS)

### CODE REVIEW

**Comment by kdenhartog** [comment]
@tplooker thoughts on best way to add samples here for it? Would you prefer to see it written in a separate file other than index.ts or add to the current sample to show deriving proof of single proofDocuments and multi proofDocuments.

File: __tests__/deriveProof.spec.ts [hunk 1]
```diff
@@ -49,4 +51,40 @@ describe("BbsBlsSignatureProof2020", () => {
     );
     expect(result).toBeDefined();
   });
+
+  it("should derive proofs from multiple proof documents", async () => {
```
Comment (tplooker) [inline __tests__/deriveProof.spec.ts:55]:
```suggestion
  it("should derive proofs from a document featuring multiple proofs", async () => {
```

File: __tests__/deriveProof.spec.ts [hunk 2]
```diff
@@ -49,4 +51,40 @@ describe("BbsBlsSignatureProof2020", () => {
...
+      testRevealDocument,
+      {
+        suite: new BbsBlsSignatureProof2020(),
+        documentLoader: customLoader
+      }
+    );
+    expect(result).toBeDefined();
+  });
+
+  it("should derive single proof from document with multiple different proofs", async () => {
```
Comment (tplooker) [inline __tests__/deriveProof.spec.ts:67]:
```suggestion
  it("should derive single proof from document with multiple different proofs", async () => {
```

**Comment by kdenhartog** [comment]
@tplooker all of the code is working correctly now with two proofs being properly derived. However, the changes to the samples won't work yet until we release the package and consume the new version.

File: src/deriveProof.ts [hunk 3]
```diff
@@ -74,13 +107,22 @@ export const deriveProof = async (
     delete compactProof[alias];
     delete compactProof["@context"];
 
+    /**
```
Comment (tplooker) [inline src/deriveProof.ts:110]: Can we create an issue to capture this and come back to it with a more permanent solution?

Comment (kdenhartog) [reply src/deriveProof.ts:110]: Done: https://github.com/mattrglobal/jsonld-signatures-bbs/issues/85

**Review by tplooker — APPROVED** [review]

**Pull request merged**